packages/docs/site/docs/01-start-here/02-overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Overview
+slug: /overview
+---
+
+# Overview
+
+WordPress Playground is an online platform that allows you to experiment and learn about WordPress without affecting your live website. It's a virtual sandbox where you can play around with different features, designs, and settings in a safe and controlled environment.
+
+## Here's how it works
+
+When you first start using WordPress Playground, you'll be provided with a separate space where you can create and customise your own WordPress website. This space is completely isolated from your actual website.
+
+## Try themes and plugins on the fly
+
+Within the WordPress Playground, you can explore various [themes](https://developer.wordpress.org/themes/getting-started/what-is-a-theme/). You can choose from a wide range of themes and see how they look on your site. You can also modify the colors, fonts, layouts, and other visual elements to create a unique design.
+In addition to themes, you can experiment with plugins too. With WordPress Playground, you can install and test different plugins to see how they work and what they can do for your site. This allows you to explore and understand the capabilities of WordPress without worrying about breaking anything.
+
+## Create content on the go
+
+Another great feature of WordPress Playground is the ability to create and edit content. You can write blog posts, create pages and add media like images and videos to your site. This helps you understand how to organize and structure your content effectively.
```

Comment:
Something may have happened as I thought I had commented here, but I figured it would be idea if we restructure this paragraph to communicate that the content disappears when finished using the Playground.

When I read it I hear that I can build my site in the Playground, but I will be quite surprised if I reload the site and all my content is missing. There are caveats, like using a persistent mode of the Playground, but I think a note here indicating that everything disappears would go a long way to prevent people spending a lot of time writing content and then discover it's all lost.